code/modules/paperwork/photography.dm

```diff
@@ -305,6 +313,8 @@
 	var/icon/res = get_base_photo_icon()
 
 	for(var/atom/A in plane_layer_sort(atoms))
+		iterator++
```

Comment:
If you're using `CHECK_TICK`, then the iterator is no longer defined

Reply:
Yeah, I didn't compile before submitting. Just fixed it, my bad